bot/resources/tags/local-file.md

```diff
@@ -10,13 +10,13 @@ with open("/this/is/path/to/my/file.png", "rb") as f:
 When using the file-like object, you have to open it in `rb` mode. Also, in this case, passing filename to it is not necessary.
 Please note that `filename` can't contain underscores. This is Discord limitation.
 
-[`discord.Embed`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed) instance has method [`set_image`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed.set_image) what can be used to set attachment as image:
+[`discord.Embed`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed) instances have a [`set_image`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed.set_image) method which can be used to set an attachment as an image:
 ```py
 embed = discord.Embed()
 # Set other fields
 embed.set_image(url="attachment://file.png")  # Filename here must be exactly same as attachment filename.
 ```
-After this, you can send embed and attachment to Discord:
```

Comment:
I did not intend for the "can" to be removed. Sorry if the instructions were unclear.

The whole line should read "After this, you can send an embed with an attachment to Discord:"